Misc/NEWS.d/next/Library/2022-09-26-21-18-47.gh-issue-60580.0hBgde.rst

```diff
@@ -0,0 +1,3 @@
+:property:`ctypes.wintypes.BYTE` definition changed from
```

Comment:
This is failing the build, maybe you need `:data:`?

Reply:
Thank you, fixed.